packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenCard/AccessTokenCard.tsx

```diff
@@ -1,136 +1,173 @@
+    />
+  )
 
-      <div className="SRC-cardContent">
-        <div className="SRC-eqHeightRow SRC-userCardName">
-          <span className={'SRC-blackText'}>{accessToken.name}</span>
-          {isExpired && (
-            <Tooltip
-              title="This token has expired. It no longer works and can only be deleted."
-              enterNextDelay={100}
+  return (
+    <Card
+      sx={{
+        my: 2,
+        height: '120px',
+        width: '100%',
+        p: 1.5,
+        pl: 4,
+        backgroundColor: isExpired ? '#fcf8e3' : 'inherit',
+      }}
+    >
+      {warningDialog}
+      <Box
+        display={'flex'}
+        alignItems={'center'}
+        justifyContent={'space-between'}
+        height={'100%'}
+      >
+        <Stack flexGrow={1} gap={1} justifyContent={'space-between'}>
+          <Typography variant={'headline3'} sx={{ fontSize: '16px' }}>
+            {accessToken.name}
+          </Typography>
+          <div>
+            <span>Permissions: </span>
+            {accessToken.scopes.map(scope => {
+              return (
+                <Tooltip
+                  key={scope}
+                  title={
+                    scopeDescriptions[scope as keyof typeof scopeDescriptions]
+                      .description
+                  }
+                >
+                  <Typography
+                    component={'span'}
+                    variant={'smallText1'}
+                    sx={{ mx: 0.25, cursor: 'default', color: 'primary.main' }}
+                  >
+                    {
+                      scopeDescriptions[scope as keyof typeof scopeDescriptions]
+                        .displayName
+                    }
+                  </Typography>
+                </Tooltip>
+              )
+            })}
+          </div>
+          <div>
+            <Typography component={'span'} variant={'smallText1'}>
+              Last used {dayjs(accessToken.lastUsed).fromNow()}
+            </Typography>
+            <Typography
+              component={'span'}
+              variant={'smallText1'}
+              color={'grey.700'}
             >
-              <span aria-hidden="true">
-                <IconSvg icon="warning" />
-              </span>
+              {' | '}
+            </Typography>
+            <Typography component={'span'} variant={'smallText1'}>
+              Created {dayjs(accessToken.createdOn).fromNow()}
+            </Typography>
+          </div>
+        </Stack>
+        <Box
+          alignSelf={'flex-start'}
+          display={'flex'}
+          gap={1}
+          alignItems={'center'}
+        >
+          {isExpired && (
+            <Tooltip title={EXPIRED_PAT_WARNING} placement={'top'}>
+              <IconButton>
+                <IconSvg
+                  icon="warning"
+                  sx={{ color: 'warning.main', fontSize: 'inherit' }}
+                  wrap={false}
+                />
+              </IconButton>
             </Tooltip>
           )}
```

Comment:
Moved this conditional warning message to be next to + aligned with the delete button, instead of awkwardly sitting next to the token's name